### ISSUE TITLE
NotImplementedError: AWS WAF captchas not supported for 2Captcha

### ISSUE THREAD

**Issue by MehrAmoonCraft** [flathunters/flathunter#656]
I pulled changes today and for immobilienscout24 Im getting : 
  File "flathunter/captcha/twocaptcha_solver.py", line 60, in solve_awswaf
    raise NotImplementedError("AWS WAF captchas not supported for 2Captcha")
NotImplementedError: AWS WAF captchas not supported for 2Captcha

What should I do?

**Comment by codders** [comment]
hi @MehrAmoonCraft ,

Thanks for pulling the latest software. That message indicates that the software has detected an AWS WAF captcha. So far support for that in 2captcha has not been implemented (though there is a pull request for it that maybe needs a bit of polish #631). So you can either try the patch in #631 and see if that resolves your issue, or you can switch to Capmonster where AWS WAF is implemented (though obviously that might mean buying more credit, and I can't promise that will solve the problem for you).

**Comment by MehrAmoonCraft** [comment]
Thanks for the answer, I tried https://github.com/flathunters/flathunter/pull/631 but still getting same error.

**Comment by codders** [comment]
Yeah - I wasn't expecting it to work without a bit of hacking. Otherwise it would already be merged. But I'll see if I get around to it over the holidays.

**Comment by codders** [comment]
I had a look today and refactored the original pr in #662 . You're welcome to try again with that patch, but for me it wasn't successful. In the meantime, Capmonster appears to be working fine.
